docs/tutorials/basics/concepts.md

```diff
@@ -0,0 +1,128 @@
+    </td>
+    <td style="border-color: white"><img src="./img/local2.png" width="530" />
+    <figcaption> Local backend when running on a SageMaker instance on cloud</figcaption>
+    </td>
+  </tr>
+</table>
+
+
+
+Local backend evaluates trials concurrently on a single machine by using subprocesses.
+SyneTune support rotating multiple GPUs on the machine, assigning the next trial to the least
+busy GPU, e.g. the GPU with the smallest amount of trials currently running. 
+
+Local backend suffers from two shortcomings; first, it limits the number of trials that can run concurrently.
+Second, it falls short in training neural network which require many GPUs, or possibly distributed across several nodes
+
+
+SageMaker backend covers this gap by executing training script in cloud and specifically in an Amazon SageMaker instance while the tuner script runs either from your local machine or from another SageMaker instance in cloud. 
+Figure x and y show these settings respectively. 
+SageMaker backend schedules one training job per trial. 
+In addition, Amazon SageMaker provides pre-build containers of ML frameworks
+(e.g., Pytorch, TensorFlow, Scikit-learn, HuggingFace) and enables users of training on cheaper preemptible machines.
+
+<table >
+  <tr>
+    <td style="border-color: white; width: 600px"><img src="./img/sm_backend1.png" width="500" />
+    <figcaption>SageMaker backend with tuner running from local machine</figcaption>
+    </td>
+    <td style="border-color: white"><img src="./img/sm_backend2.png" width="700" />
+    <figcaption> SageMaker backend with both tuner and training scripts running on cloud</figcaption>
+    </td>
+  </tr>
+</table>
+
+In SageMaker backend, Eeach trial is run as a separate SageMaker training job. This is useful for expensive workloads, 
+where all resources of an instance (or several ones) are used for training. On the other hand, training job start-up overhead is incurred for every trial.
+
+
+It is worthy to note that Syne Tune is agnostic to execution backend, 
+and users can effortlessly change between backends by modifying input argument `trial_backend` in instantiating `Tuner`. 
+See [launch_randomsearch.py](scripts/launch_randomsearch.py) for an example of local backend where `entry_point` is the training script.
+
+```python
+from syne_tune.backend import LocalBackend
+from syne_tune import Tuner
+
+trial_backend = LocalBackend(entry_point=entry_point)
+
+tuner = Tuner(
+        trial_backend=trial_backend,
+        scheduler=scheduler,
+        stop_criterion=stop_criterion,
+        n_workers=n_workers,
+    )
+```
+
+See [launch_sagemaker_backend.py](scripts/launch_sagemaker_backend.py) for an example of SageMaker backend, where
+a PyTorch container on `ml.m4.xlarge` instance is picked to run the training script (i.e.`entry_point`).
+```python
+from syne_tune.backend import SageMakerBackend
+from syne_tune import Tuner
+
+trial_backend = SageMakerBackend(
```

Comment:
This is duplicating material further down, where launcher scripts are shown and carefully introduced.